pages/home.html

```diff
@@ -10,11 +10,13 @@ <h1>Home</h1>
 <p>
 	De trainingstijden zijn 
 	<ul>
-	  <li>maandag van 18:30 tot 20:00 in Hal 1</li>
-	  <li>donderdag van 20:00 tot 21:30 in Hal 3a</li>
+	  <li>Maandag van 18:30 tot 20:00 in Hal 1</li>
```

Comment:
Actually, this is a continuation of the sentence "De trainingstijden zijn ...", so I don't agree that these are capitalised.